assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -381,6 +395,12 @@ function getInnerElement(
         gallery as there's no event when a video's poster loads. */}
         <HiddenPosterImage src={poster} onLoad={makeImageVisible} />
         {showVideoDetail && <Duration>{lengthFormatted}</Duration>}
+        {showAttribution && (
+          <Attribution
+            author={resource.attribution.author.displayName}
+            url={resource.attribution.author.url}
+          />
+        )}
```

Comment:
Given the attribution is the same for videos and images, can we bring this outside `getInnerElement`?

Reply:
Done.